pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -605,6 +605,15 @@ func applyOutlierDetection(c *cluster.Cluster, outlier *networking.OutlierDetect
 		out.MaxEjectionPercent = &wrappers.UInt32Value{Value: uint32(outlier.MaxEjectionPercent)}
 	}
 
+	if outlier.SplitExternalLocalOriginErrors {
+		out.SplitExternalLocalOriginErrors = true
+		if outlier.ConsecutiveLocalOriginFailures.GetValue() > 0 {
```

Comment:
can/should we reject in validation webhook, SplitExternalLocalOriginErrors=false and ConsecutiveLocalOriginFailures is set?

Reply:
Yes. I will add validation logic